examples/AllTests/AllTests.cpp

```diff
@@ -54,4 +54,4 @@ int main(int ac, char** av)
     return CommandLineTestRunner::RunAllTests(ac, av);
 }
 
-#include "AllTests.h"
+// #include "AllTests.h"
```

Comment:
There's an extra test in AllTests.h that isn't included in the project - HelloTest.cpp.  Looks like the file got removed when you were renaming the project file?